contributors/devel/conformance-guidelines.md

```diff
@@ -0,0 +1,102 @@
+the testcase's code directly.
+
+## Adding New Tests
+
+To include a testcase in the conformance test suite, the following
+steps must be taken:
+- the testcase must use the `framework.ConformanceIt()` function rather
+  than the `framework.It()` function
+- the testcase must include a comment immediately before the
+  `framework.ConformanceIt()` call that includes all of the required
+  metadata about the test (see the [Test Metadata](#test-metadata) section)
+- use "Promote xxx e2e test to Conformance" as template of your PR title
+- tag your PR with "/area conformance" label
+- send your PR to Sig-Architecture for review by adding "@kubernetes/sig-architecture-pr-reviews" 
+also CC the relevant Sig and Sig-Architecture
+
+The PR associated with this change will be automatically be blocked until the 
+Architecture SIG has reviewed and approved the change. The mechanism by which 
+the Architecture SIG is notified of the desired change is still TBD.
+
+## Test Metadata
+
+Each conformance test must include the following piece of metadata
+within its associated comment:
+
+- `Release`: indicates the Kubernetes release that the test was added to the
+  conformance test suite. If the test was modified in subsequent releases
+  then those releases should be included as well (comma separated)
+- `Testname`: a human readable short name of the test
+- `Description`: a detailed description of the test. This field must describe
+  the required behaviour of the Kubernetes components being tested using 
+  [RFC2119](https://tools.ietf.org/html/rfc2119) keywords. This field
+  is meant to be a "specification" of the tested Kubernetes features, as
+  such, it must be detailed enough so that readers can fully understand
+  the aspects of Kubernetes that are being tested without having to read
+  the test's code directly. Additionally, this test should provide a clear
+  distinction between the parts of the test that are there for the purpose
+  of validating Kubernetes rather than simply infrastructure logic that
+  is necessary to setup, or clean up, the test.
+
+## Sample Conformance Test
+
+The following snippet of code shows a sample conformance test's metadata:
+
+```
+/*
+  Release : v1.9
+  Testname: Kubelet: log output
+  Description: By default the stdout and stderr from the process being
+  executed in a pod MUST be sent to the pod's logs.
+*/
+framework.ConformanceIt("it should print the output to logs", func() {
+  ...
+})
+```
+
+The corresponding portion of the Conformance Document would then look
+like this:
+
+>
+> ## [Kubelet: log output](https://github.com/kubernetes/kubernetes/tree/release-1.9/test/e2e_node/kubelet_test.go#L47)
+> 
+> Release : v1.9
+> 
+> By default the stdout and stderr from the process being executed in a pod MUST be sent to the pod's logs.
+
+
+Conformance test results can also be viewed from the testgrid, details can be found [here](https://docs.google.com/document/d/1lGvP89_DdeNO84I86BVAU4qY3h2VCRll45tGrpyx90A/edit#).
```

Comment:
Consider modifying the last line as follows:

Conformance test results, by provider and releases, can be viewed in the federated [Conformance TestGrid dashboard](https://k8s-testgrid.appspot.com/conformance-all).  If you wish to contribute conformance test results for your provider, please follow this [on-boarding document](https://docs.google.com/document/d/1lGvP89_DdeNO84I86BVAU4qY3h2VCRll45tGrpyx90A/edit#).